web/modules/custom/dbcdk_community/src/Form/SettingsForm.php

```diff
@@ -44,14 +44,37 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $config = $this->config('dbcdk_community.settings');
 
     // Community service host url.
-    $form['community_service_url'] = array(
-      'community_service_url' => array(
-        '#type' => 'textfield',
-        '#title' => $this->t('Community Service Url'),
-        '#default_value' => $config->get('community_service_url'),
-        '#description' => $this->t('The Community Service Url until the endpoints start. Example: %example', array('%example' => 'https://dbcdk-community.dk/api/')),
-      ),
+    $form['community_service_url'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Community Service Url'),
+      '#default_value' => $config->get('community_service_url'),
+      '#description' => $this->t('The Community Service Url until the endpoints start. Example: %example', array('%example' => 'https://dbcdk-community.dk/api/')),
+    ];
+
+    // Vertical tab for community site settings.
+    $form['community_site'] = array(
```

Comment:
If you find any reason to "correct" this, feel free to do so, but I really dont mind. But I can see that you've changed "community service url" and "community service debug" to use `[]` instead of `array()` when you removed the unnecessary nesting.

In short: you're using `array()` here. Change it if you feel like it.